Tests are failing due to `go list -m -f {{.Dir}}` returning two entries

```shell
$ go test
2024/01/19 20:57:42 running: go list -m -f {{.Dir}}
--- FAIL: TestLoadRunScript (0.00s)
    parse_script_test.go:51: run script not found for qf104: stat /Users/meling/work/quickfeed
        /Users/meling/work/quickfeed/kit/testdata/courses/qf104-2022/tests/scripts/run.sh: no such file or directory
    parse_script_test.go:54: run script is empty
    parse_script_test.go:59: run script not found for qf104: stat /Users/meling/work/quickfeed
        /Users/meling/work/quickfeed/kit/testdata/courses/qf104-2022/tests/scripts/run.sh: no such file or directory
    parse_script_test.go:62: run script is empty
--- FAIL: TestParseTestRunnerScript (0.00s)
    parse_script_test.go:85: run script not found for DAT320: stat /Users/meling/work/quickfeed
        /Users/meling/work/quickfeed/kit/testdata/courses/qf104-2022/tests/scripts/run.sh: no such file or directory
--- FAIL: TestParseBadTestRunnerScript (0.00s)
    parse_script_test.go:135: err = 'run script not found for DAT320: stat /Users/meling/work/quickfeed
        /Users/meling/work/quickfeed/kit/testdata/courses/qf104-2022/tests/scripts/run.sh: no such file or directory', want 'failed to parse run script for assignment lab4-bad-run-script in https://github.com/qf104-2022/tests: empty run script'
    parse_script_test.go:145: err = 'run script not found for DAT320: stat /Users/meling/work/quickfeed
        /Users/meling/work/quickfeed/kit/testdata/courses/qf104-2022/tests/scripts/run.sh: no such file or directory', want 'failed to parse run script for assignment lab5-bad-run-script in https://github.com/qf104-2022/tests: no docker image specified in run script'
FAIL
exit status 1
FAIL	github.com/quickfeed/quickfeed/ci	1.106s
```
These tests are failing because:
```sh
$ go list -m -f {{.Dir}}
/Users/meling/work/quickfeed
/Users/meling/work/quickfeed/kit
```
is returning two entires because we added `go.work`, which solves other problems.

--- a/internal/env/load.go
+++ b/internal/env/load.go
@@ -83,5 +83,5 @@ func Load(filename string) error {
 
 func ignore(line string) bool {
 	trimmedLine := strings.TrimSpace(line)
-	return len(trimmedLine) == 0 || strings.HasPrefix(trimmedLine, "#")
+	return trimmedLine == "" || strings.HasPrefix(trimmedLine, "#")
 }

--- a/internal/env/load.go
+++ b/internal/env/load.go
@@ -10,14 +10,17 @@ import (
 	"github.com/quickfeed/quickfeed/kit/sh"
 )
 
-const dotEnvPath = ".env"
+const (
+	dotEnvPath          = ".env"
+	quickfeedModulePath = "github.com/quickfeed/quickfeed"
+)
 
 var quickfeedRoot string
 
 func init() {
 	quickfeedRoot = os.Getenv("QUICKFEED")
 	if quickfeedRoot == "" {
-		out, err := sh.Output("go list -m -f {{.Dir}}")
+		out, err := sh.Output("go list -m -f {{.Dir}} " + quickfeedModulePath)
 		if err != nil {
 			log.Fatalf("Failed to set QUICKFEED variable: %v", err)
 		}

